OpenTibia/Assets/Scripts/UI/Legacy/ContextMenuBase.cs

```diff
@@ -62,7 +62,7 @@ public class ContextMenuBase : Core.Components.Base.AbstractComponent
         private void InternalCreateSeparatorItem() {
             var separatorGO = Instantiate(OpenTibiaUnity.GameManager.HorizontalSeparator, transform);
             var layoutElement = separatorGO.AddComponent<UnityUI.LayoutElement>();
-            layoutElement.minWidth = 120;
+            layoutElement.minWidth = 80;
```

Comment:
Is that all what it takes? It's no dynamic in any way? I'm still thinking about mobile platforms and supposedly we will have to deal with different resolutions, but let's leave it as it is for now.

Reply:
So why don't we set it to 0? :)

Reply:
That was suggestion from more experienced Unity users. :)

Reply:
Let's leave it as it is now then. I'm just afraid it will result in regression in some other part of the system, as we don't really know what is the purpose of having this minimum value set to value > 0.